pyproject.toml

```diff
@@ -0,0 +1,84 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+description = "A Python client library for connecting securely to your Google Cloud AlloyDB instances."
+name = "google-cloud-alloydb-connector"
+authors = [{ name = "Google LLC", email = "googleapis-packages@google.com" }]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
```

Comment:
This is actually using the [legacy license declaration](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#legacy-license-declaration), you will want to update this or else you will have broken builds and deprecation warnings as of late.

See cloud sql https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/pull/1249